spec/mailers/previews/candidate_mailer_preview.rb

```diff
@@ -104,37 +104,37 @@ def chase_candidate_decision_with_multiple_offers
   def new_offer_single_offer
     conditions = [FactoryBot.build(:offer_condition, text: 'DBS check'),
                   FactoryBot.build(:offer_condition, text: 'Pass exams')]
-    application_choice = FactoryBot.build_stubbed(:application_choice,
-                                                  :with_offer,
-                                                  application_form: application_form,
-                                                  course_option: course_option,
-                                                  offer: FactoryBot.build_stubbed(:offer, conditions: conditions),
-                                                  current_course_option: course_option,
-                                                  decline_by_default_at: 10.business_days.from_now)
+    application_choice = FactoryBot.build(:application_choice,
+                                          :with_offer,
+                                          application_form: application_form,
+                                          course_option: course_option,
+                                          offer: FactoryBot.build_stubbed(:offer, conditions: conditions),
```

Comment:
```suggestion
                                          offer: FactoryBot.build(:offer, conditions: conditions),
```

Reply:
and a few other places :D

Reply:
I can change the nested objects as well, but they don't cause any problems as using `build` is only required on application_choice so it skips the `:stub` callback where we use `allow`

Reply:
@cwrw done!

Reply:
thanks, was mostly for consistency but also ok to keep as is. Realised there are a few other instances this is also breaking, I think the following need to also use `build` such as `application_rejected_by_default_offers_only, application_rejected_one_offer_one_awaiting_decision, application_rejected_by_default_one_offer_one_awaiting_decision, application_rejected_offers_only, application_rejected_one_offer_one_awaiting_decision`